src/login_helper.js

```diff
@@ -61,6 +66,347 @@ function filterPastAndUpcomingTrips(travelerTrips) {
+  let tripStartLabel = document.createElement("label");
+  tripStartLabel.setAttribute("for", "trip-start");
+  appendToRequestForm(tripStartLabel);
+  let today = new Date().toLocaleDateString();
+  let startDateInput = document.createElement("input");
+  startDateInput.setAttribute("type", "date");
+  startDateInput.setAttribute("id", "start");
+  startDateInput.setAttribute("name", "trip-start");
+  startDateInput.setAttribute("value", `${today}`);
+  startDateInput.setAttribute("min", `${today}`);
+  startDateInput.setAttribute("max", "2021-12-31");
+  appendToRequestForm(startDateInput);
+  let tripEndLabel = document.createElement("label");
+  tripEndLabel.setAttribute("for", "trip-end");
+  appendToRequestForm(tripEndLabel);
+  let endDateInput = document.createElement("input");
+  endDateInput.setAttribute("type", "date");
+  endDateInput.setAttribute("id", "end");
+  endDateInput.setAttribute("name", "trip-end");
+  endDateInput.setAttribute("value", `${today}`);
+  endDateInput.setAttribute("min", `${today}`);
+  endDateInput.setAttribute("max", "2023-12-31");
+  appendToRequestForm(endDateInput);
+}
+
+function prepareTripRequestData(destinations) {
+  let tripRequestData = [];
+  let tripRequestDestination = document.getElementById("destination").value;
+  let numberOfTravelers = document.getElementById("number-of-travs").value;
+  let tripRequestStartDate = document.getElementById("start").value;
+  let tripRequestEndDate = document.getElementById("end").value;
+  let i;
+  for (i = 0; i < destinations.length; i++) {
+    if (tripRequestDestination === destinations[i].destination) {
+      let destinationIdentifier = destinations[i].id;
+      tripRequestData.push(destinationIdentifier);
+    }
+  }
+  tripRequestData.push(
+    "Destination: " + tripRequestDestination,
+    "Number of Travelers: " + numberOfTravelers,
+    "Trip Start Date: " + tripRequestStartDate,
+    "Trip End Date:" + tripRequestEndDate
+  );
+  return tripRequestData;
+}
+function renderDestinationImage(destinationInfo) {
+  let destinationImage = destinationInfo.image;
+  var img = document.createElement("img");
+  img.className = "destination-image";
+  img.src = `${destinationImage}`;
+  document.querySelectorAll("section")[3].appendChild(img);
+}
+
+function getValuesFromInputs() {
+  let values = [];
+  let tripRequestDestination = document.getElementById("destination").value;
+  let tripRequestStartDate = document.getElementById("start").value;
+  let tripRequestEndDate = document.getElementById("end").value;
+  let numberOfTravelers = document.getElementById("number-of-travs").value;
+  values.push(
+    tripRequestDestination,
+    tripRequestStartDate,
+    tripRequestEndDate,
+    numberOfTravelers
+  );
+  return values;
+}
+
+function calculateTripRequestCosts(destination) {
+  let costs = [];
+  let values = getValuesFromInputs();
+  let numberOfTravelers = values[3];
+  let tripRequestStartDate = values[1];
+  let tripRequestEndDate = values[2];
```

Comment:
you don't really need these lines if your getValuesFromInputs(); is returning an object